packages/KTextArea/KTextArea.vue

```diff
@@ -117,6 +109,21 @@ export default {
       delete listeners['input']
 
       return listeners
+    },
+    charLimitExceeded () {
+      return !this.disableCharacterLimit && this.currValue.length > this.characterLimit
+    }
+  },
+  watch: {
+    charLimitExceeded (newval, oldval) {
+      if (newval !== oldval) {
+        this.$emit('char-limit-exceed', {
```

Comment:
Incorrect event name, should match the docs: `char-limit-exceeded`

Reply:
Thank you! Fixed.